plugin/src/main/java/io/jenkins/plugins/analysis/core/model/IssuesDetail.java

```diff
@@ -502,7 +504,7 @@ else if ("origin".equals(propertyName)) {
         else {
             propertyFormatter = Function.identity();
         }
-        return new PropertyStatistics(report, propertyName, propertyFormatter);
+        return new PropertyStatistics(report, propertyName, propertyFormatter, getLabelProvider().getAgeBuilder(owner,  getUrl()));
```

Comment:
It would rather make sense to use the field `newIssues` to determine the new issues. Using the age builder seems more like a hack. Using the age would be also inconsistent for the first build where we do not have any new issues.

Reply:
That makes more sense. Thanks for the suggestion!